File: airbyte-integrations/connectors/destination-s3/src/test-integration/java/io/airbyte/integrations/destination/s3/S3CsvAssumeRoleDestinationAcceptanceTest.java
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */

package io.airbyte.integrations.destination.s3;

import com.fasterxml.jackson.databind.JsonNode;
import io.airbyte.cdk.integrations.destination.s3.S3BaseCsvDestinationAcceptanceTest;
import java.util.Map;

public class S3CsvAssumeRoleDestinationAcceptanceTest extends S3BaseCsvDestinationAcceptanceTest {

  @Override
  protected JsonNode getBaseConfigJson() {
    return S3DestinationTestUtils.getAssumeRoleConfig();
  }

  @Override
  public Map<String, String> getConnectorEnv() {
    return S3DestinationTestUtils.getAssumeRoleInternalCredentials();
  }

}

File: airbyte-integrations/connectors/destination-s3/src/test-integration/kotlin/io/airbyte/integrations/destination/s3/S3CsvAssumeRoleDestinationAcceptanceTest.kt
/*
 * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 */
package io.airbyte.integrations.destination.s3

import com.fasterxml.jackson.databind.JsonNode
import io.airbyte.cdk.integrations.destination.s3.S3BaseCsvDestinationAcceptanceTest

class S3CsvAssumeRoleDestinationAcceptanceTest : S3BaseCsvDestinationAcceptanceTest() {
    override val baseConfigJson: JsonNode
        get() = S3DestinationTestUtils.assumeRoleConfig

    override fun getConnectorEnv(): Map<String, String> {
        return S3DestinationTestUtils.assumeRoleInternalCredentials
    }
}
